src/infra/app-deployment.yaml

```diff
@@ -11,8 +11,8 @@ spec:
     metadata:
       labels:
         app: user
-        sidecar.istio.io/inject: "true"
-        auth: jwt
+        #sidecar.istio.io/inject: "true"
```

Comment:
@eyakauleva why do you remove sidecar injection?

Reply:
@h1alexbel cause my laptop can't handle both istio and strimzi

Reply:
@eyakauleva it's sad(

Reply:
@h1alexbel It seems you forgot to approve the PR